vpr/src/place/place_constraints.cpp

```diff
@@ -82,29 +82,69 @@ PartitionRegion constrained_macro_locs(const t_pl_macro& pl_macro) {
 
                 t_pl_loc min_pl_loc(reg_rect.xmin(), reg_rect.ymin(), block_regions[i].get_sub_tile());
 
-                t_pl_loc modified_min_pl_loc = min_pl_loc + offset;
```

Comment:
See if you can refactor this by creating an offset helper function for vtr::Rect